docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -161,12 +164,12 @@ quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus/
 quarkus.oidc.client-id=quarkus-app
 
 # This is a key which will be used to retrieve a secret from the map of credentials returned from CredentialsProvider
-quarkus.oidc.credentials.jwt.secret-provider.key=mysecret-key
+quarkus.oidc.credentials.jwt.secret-provider.key=mysecret-key.
 # Set it only if more than one CredentialsProvider can be registered
-quarkus.oidc.credentials.jwt.secret-provider.name=oidc-credentials-provider
+quarkus.oidc.credentials.jwt.secret-provider.name=oidc-credentials-provider.
 ----
 
-Example of `private_key_jwt` with the PEM key file, and where the signature algorithm is RS256:
+Example of `private_key_jwt` with the Privacy Enhanced Mail (PEM) key file, and where the signature algorithm is RS256:
```

Comment:
```suggestion
Example of `private_key_jwt` with the PEM key file, and where the signature algorithm is RS256:
```

We don't really expand PEM like that, it would probably confuse some users even if it is correct

Reply:
Reverted this change per your suggestion.